cmd/restic/exclude.go

```diff
@@ -88,6 +88,16 @@ func rejectByPattern(patterns []string) RejectByNameFunc {
 	}
 }
 
+// Same as `rejectByPattern` but case insensitive.
+func rejectByInsensitivePattern(patterns []string) RejectByNameFunc {
+	for index, path := range patterns {
+		patterns[index] = strings.ToLower(path)
+	}
+	return func(item string) bool {
+		return rejectByPattern(patterns)(strings.ToLower(item))
```

Comment:
For every file that is to be tested during backup, `rejectByPattern()` creates a new function that is instantly evaluated. What about creating the function just once, outside the closure, and then use it inside?
```suggestion
	rejFunc := rejectByPattern(patterns)
	return func(item string) bool {
		return rejFunc(strings.ToLower(item))
```